setup.py

```diff
@@ -32,9 +32,6 @@ def read(filename):
 if sys.version_info < (3, 4):
```

Comment:
shouldn't this also be removed, since support now starts at >= 3.4?

Reply:
Indeed thanks for pointing it out.

Reply:
What about Python 2.7?

Reply:
This one has not reached EOL and is still widely used. So I don't believe, we (pyvisa maintainers) will drop it before 2020-2021 or even a bit later. That said I won't fight to keep it longer than necessary as it increases the maintenance burden.

Reply:
Yes, still big numbers for 2.7 at https://github.com/pyvisa/pyvisa/issues/295#issuecomment-349346466. 

Python 2.7 doesn't have the `enum` module, but this `enum34` package backports it and is needed for 2.7.

It's probably not worth dropping 2.7 just to remove this dependency.

Reply:
I should have been more careful in checking that. I reverted my commit. Sorry @hugovk.

Reply:
I may edit the check later to avoid mentioning Python 3.4 to avoid future confusions.

Reply:
Good idea, a simple Py2 vs. Py3 check would be better!